splinepy/io/json.py

```diff
@@ -0,0 +1,153 @@
+"""
+Export splines in custom json format
+"""
+
+import base64
+import json
+import logging
+import numpy as np
+
+def load(fname):
+    """
+    Loads splines from json file. Returns dict-splines
+
+    Parameters
+    -----------
+    fname: str
+
+    Returns
+    --------
+    spline_list: list
+    """
+    # Import data from file into dict format
+    jsonbz = json.load(open(fname, "r"))
+
+    spline_list = []
+    base64encoding = jsonbz["Base64Encoding"]
+
+    # Init return type
+    return_dict = {
+        "Bezier" : [],
+        "NURBS" : [],
+        "BSpline" : []
+    }
+
+    for jbz in jsonbz["SplineList"]:
+
+        # Convert Base64 str into np array
+        if base64encoding:
+            jbz["control_points"] = \
+                np.frombuffer(
+                  base64.b64decode(
+                    jbz["control_points"].encode('ascii')
```

Comment:
first encode to utf-8 for export and encode again to ascii?

Reply:
Yes, the string must be interpreted as ascii so that the char to byte conversion matches the base64 string conversion